executor/executor_xapi.go

```diff
@@ -170,6 +170,7 @@ type lookupTableTask struct {
 	rows    []*Row
 	cursor  int
 	done    bool
+	respCh  chan error
```

Comment:
The name confuses me very much. So I think we should add comment for it.
"respCh" make me believe that the task's response data will be returned from this channel.